.github/workflows/e2e.yml

```diff
@@ -141,20 +141,36 @@ jobs:
 
       - name: Install ilab
         run: |
-          export PATH="/home/runner/.local/bin:/usr/local/cuda/bin:$PATH"
-          python3 -m venv venv
+          export PATH="/home/ec2-user/.local/bin:/usr/local/cuda/bin:$PATH"
+          python3.11 -m venv --upgrade-deps venv
           . venv/bin/activate
           sed 's/\[.*\]//' requirements.txt > constraints.txt
-          python3 -m pip cache remove llama_cpp_python
-          CMAKE_ARGS="-DLLAMA_CUBLAS=on" python3 -m pip install --no-binary llama_cpp_python -c constraints.txt llama_cpp_python
-          # needed for --4-bit-quant option to ilab model train
-          python3 -m pip install bitsandbytes
-          python3 -m pip install .
+          python3.11 -m pip cache remove llama_cpp_python
+          CMAKE_ARGS="-DLLAMA_CUBLAS=on" python3.11 -m pip install --force-reinstall --no-binary llama_cpp_python -c constraints.txt llama_cpp_python
+          python3.11 -m pip install bitsandbytes
+
+          # TODO This should be added to instructlab-training
+          python3.11 -m pip install packaging wheel
+
+          python3.11 -m pip install instructlab-training[cuda]
+
+          python3.11 -m pip install .
 
       - name: Run e2e test
         run: |
+          export PYTORCH_CUDA_ALLOC_CONF=garbage_collection_threshold:0.6,max_split_size_mb:128
           . venv/bin/activate
-          ./scripts/basic-workflow-tests.sh -cm
+          python3.11 -m pip uninstall -y deepspeed
+
+          python3.11 -m pip uninstall -y nvidia-nccl-cu12
```

Comment:
what's going on here?  the other workflow didn't do this

Reply:
sorry, forgot to remove changes from the base e2e.yml